extensions/ql-vscode/src/remote-queries/sarif-processing.ts

```diff
@@ -163,17 +169,45 @@ export function tryGetRule(
 }
 
 function getCodeSnippet(
+  contextRegion?: sarif.Region,
```

Comment:
I spent a long time looking at the change to ordering of these parameter and whether we'd need to update the call-sites. I now agree it's a no-op because when we do `actualRegion = contextRegion ?? region` below the name change is consistent there. So I believe this is all fine, but it's a little confusing at first glance when reviewing.